.env.example

```diff
@@ -0,0 +1,35 @@
+# Firestore Collections
```

Comment:
We do have a script to generate it. https://github.com/astronomer/ask-astro/blob/main/scripts/local_dev.py I'm thinking of merging it into https://github.com/astronomer/ask-astro/tree/main/tasks. @pankajastro @sunank200  WDYT?